app/models/github_repo.rb

```diff
@@ -27,31 +27,8 @@ def self.upsert(user, **params)
   end
 
   def self.update_to_latest
-    where("updated_at < ?", 26.hours.ago).includes(:user).find_each do |repo|
-      user = repo.user
-      next unless user
-
-      begin
-        client = Github::OauthClient.for_user(user)
-        fetched_repo = client.repository(repo.info_hash[:full_name])
-
-        repo.update!(
-          github_id_code: fetched_repo.id,
-          name: fetched_repo.name,
-          description: fetched_repo.description,
-          language: fetched_repo.language,
-          fork: fetched_repo.fork,
-          bytes_size: fetched_repo.size,
-          watchers_count: fetched_repo.watchers,
-          stargazers_count: fetched_repo.stargazers_count,
-          info_hash: fetched_repo.to_hash,
-        )
-        repo.user&.touch(:github_repos_updated_at)
-      rescue Github::Errors::NotFound
-        repo.destroy
-      rescue StandardError
```

Comment:
I removed this when I transferred the code to the worker. I am not sure what it was there for but we will find out if we see workers fail.